src/client/Client.cc

```diff
@@ -189,18 +189,37 @@ void Client::_assign_faked_ino(Inode *in)
     it = free_faked_inos.lower_bound(last_used_faked_ino + 1);
   }
   ceph_assert(it != free_faked_inos.end());
-  if (last_used_faked_ino < it.get_start()) {
+  if (last_used_faked_ino < (it.get_start() + 1024)) {
     ceph_assert(it.get_len() > 0);
-    last_used_faked_ino = it.get_start();
+    last_used_faked_ino = it.get_start() + 1024;
   } else {
     ++last_used_faked_ino;
-    ceph_assert(it.get_start() + it.get_len() > last_used_faked_ino);
+    ceph_assert(it.get_start() + 1024 + it.get_len() > last_used_faked_ino);
```

Comment:
we should remove the skipped inos from the interval set

Reply:
thanks for your reviews.
_assign_faked_ino() and _assign_faked_root share free_faked_inos. If I remove the skipped inos here, will _release_faked_root not be allocated?

my modification here is: assign last_used_faked_ino after 2048, and it is normal from the result.
why is last_used_faked_ino's initialization 0, not 1024, or it can be set to 2048?